src/cut/src/abc_library_factory.cpp

```diff
@@ -680,4 +686,21 @@ bool AbcLibrary::IsConstCell(const std::string& cell_name)
   return IsConst1Cell(cell_name) || IsConst0Cell(cell_name);
 }
 
+static bool abc_initialized = false;
+
+void abcInit()
+{
+  if (!abc_initialized) {
+    abc::Abc_Start();
+    abc_initialized = true;
+  }
```

Comment:
This should really use call_once https://en.cppreference.com/w/cpp/thread/call_once.html